### PR TITLE
bumped all deps

### PR DESCRIPTION
**Pull request by romni**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -43,13 +43,12 @@ dependencies {
...
-    compile 'io.projectreactor.ipc:reactor-ipc:0.6.1.RELEASE'
-    compile 'io.projectreactor.ipc:reactor-netty:0.6.2.RELEASE'
-    compile 'org.reactivestreams:reactive-streams:1.0.0'
+    compile 'io.projectreactor:reactor-core:3.1.2.RELEASE'
+    compile 'io.projectreactor.ipc:reactor-ipc:0.6.2.RELEASE'
+    compile 'io.projectreactor.ipc:reactor-netty:0.7.2.RELEASE'
 
     // configuration
-    compile 'com.typesafe:config:1.3.1'
+    compile 'com.typesafe:config:1.3.2'
```
Comment (thumbnail) [inline build.gradle]: Let's remove this one

File: build.gradle [hunk 2]
```diff
@@ -59,14 +58,14 @@ dependencies {
     compile 'com.github.dblock:oshi-core:3.4.0'
 
     // db
-    compile 'org.influxdb:influxdb-java:2.5'
+    compile 'org.influxdb:influxdb-java:2.8'
```
Comment (thumbnail) [inline build.gradle:46]: Let's remove this one, and add it subra-examples

File: src/main/java/io/leonis/zosma/ipc/io/ConfigPublisher.java [hunk 3]
```diff
@@ -1,20 +1,35 @@
...
 
 /**
+ * The Class ConfigPublisher.
+ *
+ * This class represents a {@link Publisher} of {@link Config}.
+ *
  * @author Jeroen de Jong
  */
 @AllArgsConstructor
 public final class ConfigPublisher implements Publisher<Config> {
```
Comment (thumbnail) [inline src/main/java/io/leonis/zosma/ipc/io/ConfigPublisher.java]: I think we should remove this class with the dependency on typesafe.config; should we do it in a seperate PR ?

File: src/main/java/io/leonis/zosma/ipc/ip/TCPPublisher.java [hunk 4]
```diff
@@ -25,7 +25,7 @@
 
   @Override
   public void subscribe(final Subscriber<? super DatagramPacket> subscriber) {
-    TcpServer.create(opts -> opts.option(ChannelOption.SO_KEEPALIVE, true).listen(this.port))
+    TcpServer.create(opts -> opts.option(ChannelOption.SO_KEEPALIVE, true).port(this.port))
```
Comment (thumbnail) [inline src/main/java/io/leonis/zosma/ipc/ip/TCPPublisher.java:29]: Could you update the javadoc for port as well ?

Comment (romni) [reply src/main/java/io/leonis/zosma/ipc/ip/TCPPublisher.java:29]: nothing changed? (except for the api)

Comment (thumbnail) [reply src/main/java/io/leonis/zosma/ipc/ip/TCPPublisher.java:29]: Yeah but don't you think that `The port to listen on.` seems a little strange?

File: build.gradle (hunk 2 above)
Comment (thumbnail) [reply build.gradle:46]: Nevermind, i noticed we have a influx subscriber as well.

**Review by thumbnail — APPROVED** [review]

**Pull request merged**